### DISCUSSION TITLE
Modbus error not started

### DISCUSSION THREAD

**Discussion by nex2000** [frangoteam/FUXA#633] [Q&A]
I have installed fuxa on windows 10 , all work fine, but modbus com1 scan port not start . I got this error . Why this happen?

2022-12-07T14:44:01.176Z [ERR]  'Modbus' connect failed! Error: Could not locate the bindings file. Tried:
 → C:\Users\Utente\AppData\Roaming\npm\node_modules\@frangoteam\fuxa\_pkg\@serialport\bindings\build\bindings.node

Reagards
NEX

**Comment by nex2000** [comment]
I solve it , the nodejs release was 12.20 instead 12.22

**Comment by danilliu-sudo** [comment]
dear nex2000, which subversion does the nodejs 12.20 need？ I have the same problem, and tried several 12.22 and 12.20 version, but can't solve the problem. Thanks.

**Reply by nex2000** [reply]
Dear danilliu

My problem was a wrong nodejs version , I remeber you after  complete installation you must install modbus and serial port.  I can assure you that the serial modbus work very fine , I tested it with 200 ms scan rate. Regards
